lib/plugins/particles.js

```diff
@@ -0,0 +1,7 @@
+module.exports = inject
+
+function inject (bot) {
+  bot._client.on('world_particles', (packet) => {
+    bot.emit('particle', packet)
```

Comment:
we dont want to give the whole packet, just the data

Reply:
but data usually is undefined (if we are strictly peaking what the event emits)